### ISSUE TITLE
Distortion of proximal gastric anatomy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10051]
**Preferred term label:**
Distortion of proximal gastric anatomy

**Synonyms**
Difficulty with intubation of the stomach and pylorus

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK448064/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I see the mention of this here https://www.ncbi.nlm.nih.gov/books/NBK448064/, but there is no actual description of the distortion, and so I am not sure that it would be obvious when to use the HPO if we make one. I cannot find more information in PubMed. I think it would be good to have at least one publication with an image of what the distortion looks like, otherwise, I think this is probably coming from some case report and would not be precise enough to make an HPO term?

**Comment by MickeySegal** [comment]
In an article "Gastric Volvulus, Borchardt's Triad, and Endoscopy"
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3072542/
"Endoscopic diagnosis can reveal a tortuous appearance of the stomach and difficulty or inability to reach the pylorus."

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000987
- label: Tortuous appearance of the stomach
- parents: [Abnormal stomach morphology (HP:0002577)](https://hpo.jax.org/app/browse/term/HP:0002577)
- pmid: 
- def: Abnormal twisting or winding of the stomach.
- comment: This feature can be seen with gastric volvulus and cam imply that it is difficult to reach the pylorus with gastric endoscopy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10051
- synonyms:
